smartsim/entity/files.py

```diff
@@ -130,3 +130,122 @@ def _check_path(self, file_path):
         if path.isdir(full_path):
             return full_path
         raise SSConfigError(f"File or Directory {file_path} not found")
+
+
+class TaggedFilesHierarchy:
+    """The TaggedFilesHierarchy class maintains a list of files and a list of
+    child TaggedFilesHierarchy instances. Each instance describes how a
+    directory structure of tagged files.
```

Comment:
Lets shorten these comment lines a bit (20 characters) so that it reads nicer on VSCode snippet and docs.

Reply:
resolved in https://github.com/CrayLabs/SmartSim/pull/119/commits/a815885a9b8d49eab4552f1801a6f1631de3f215